### DISCUSSION TITLE
Optional chaining and newline error

### DISCUSSION THREAD

**Discussion by jira-zz** [eslint/eslint#19009] [General]
This is ok:
```
const rows = data?.filter(....
```

So is this:
```
const rows = data
?.filter(....
```

Here eslint gives an error:
```
const rows = data?
.filter(....
```

Is this expected behavior?

**Comment by mdjermanovic** [comment]
> Here eslint gives an error:
> 
> ```
> const rows = data?
> .filter(....
> ```
> 
> Is this expected behavior?

Yes, if it reports a parsing error. `?.` is an operator. `?` and `.` separated by whitespace is invalid syntax.
